nixos/modules/services/monitoring/loki.nix

```diff
@@ -99,9 +99,16 @@ in {
         conf = if cfg.configFile == null
                then prettyJSON cfg.configuration
                else cfg.configFile;
+        validateConfig = file:
+        pkgs.runCommand "validate-loki-conf" {
+          nativeBuildInputs = [ pkgs.grafana-loki ];
```

Comment:
This should use `cfg.package` not `pkgs.grafana-loki` :(

Reply:
I'll provide the PR to fix this, I should have seen this.

Reply:
#312294